packages/upload-core/upload.test.js

```diff
@@ -42,14 +35,6 @@ describe('upload.core', () => {
     new Upload(file, options); // eslint-disable-line
   });
 
-  it('should throw error for invalid file type', () => {
```

Comment:
these no longer fail to rejected immediately, can add test cases back

Reply:
we should definitely add test around the new interface